src/dstr-binding/obj-ptrn-rest-nested-obj.case

```diff
@@ -1,25 +0,0 @@
-// Copyright (C) 2017 Caio Lima. All rights reserved.
-// This code is governed by the BSD license found in the LICENSE file.
-
-/*---
-desc: >
-    When DestructuringAssignmentTarget is an object literal, it should be parsed
-    parsed as a DestructuringAssignmentPattern and evaluated as a destructuring
-    assignment.
```

Comment:
It's a little late for me to be asking now, but: Why is this desc talking about destructuring assignment? It doesn't look like it has to do with that; the test is all about destructuring binding.